osu.Game.Rulesets.Catch/CatchRuleset.cs

```diff
@@ -29,6 +32,42 @@ public class CatchRuleset : Ruleset
             new KeyBinding(InputKey.Shift, CatchAction.Dash),
         };
 
+        public override IEnumerable<BeatmapStatistic> GetBeatmapStatistics(WorkingBeatmap beatmap)
+        {
+            IEnumerable<HitObject> hitObjects = beatmap.Beatmap.HitObjects;
+            IEnumerable<HitObject> fruits = hitObjects.Where(c => !(c is IHasEndTime));
+            IEnumerable<HitObject> trails = hitObjects.Where(s => s is IHasCurve);
+            IEnumerable<HitObject> rain = hitObjects.Where(s => s is IHasEndTime && !(s is IHasCurve));
+
+            return new[]
+            {
+                new BeatmapStatistic
+                {
+                    Name = @"Object Count",
+                    Content = hitObjects.Count().ToString(),
+                    Icon = FontAwesome.fa_circle
+                },
+                new BeatmapStatistic
+                {
+                    Name = @"Fruit Count",
+                    Content = fruits.Count().ToString(),
+                    Icon = FontAwesome.fa_circle_o
+                },
+                new BeatmapStatistic
+                {
+                    Name = @"Trail Count",

```

Comment:
where are you getting these terms from?

Reply:
> Horizontal-axis placement determines the falling point of the fruits(hit circle) or fruit trails(slider).

From here. https://osu.ppy.sh/help/wiki/Game_Modes/osu!catch

Reply:
If you have a chance, can you give the correct terms for the objects in the other modes?